.changeset/clever-needles-count.md

```diff
@@ -0,0 +1,8 @@
+---
+'@twilio-paste/box': minor
+'@twilio-paste/text': minor
+'@twilio-paste/style-props': minor
```

Comment:
I actually buy this now, I think you're right, it's technically a minor even though the package is a major bump. It doesn't change the type signatures to the Paste consumer as we don't really directly expose it.